src/commands/console.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const AppConfig = require('../models/app_configuration.js');
+const Domain = require('../models/domain.js');
+const handleCommandStream = require('../command-stream-handler.js');
+const Logger = require('../logger.js');
+const OpenBrowser = require('../open-browser.js');
+
+function console (api, params) {
```

Comment:
Same here :smile:

I'm afraid this naming could confuse people with the already existing global variable `console`.

* Could you rename this?
* Maybe something like `consoleModule` or `consoleFn`